spellchecker/utils.py

```diff
@@ -63,19 +63,19 @@ def function_wrapper(*args, **kwargs):
     return decorator_wrapper
 
 
-def ensure_unicode(_str: KeyT, encoding: str = "utf-8") -> str:
+def ensure_unicode(value: KeyT, encoding: str = "utf-8") -> str:
```

Comment:
pylint 2.9.6 thinks that an underscored parameter is meant to be ignored

renamed for clarity